build.gradle

```diff
@@ -29,17 +27,15 @@ apply plugin: 'kotlin'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'com.github.ben-manes.versions'
 apply plugin: 'com.vanniktech.code.quality.tools'
-apply plugin: 'com.vanniktech.android.junit.jacoco'
 apply plugin: "com.vanniktech.maven.publish"
-apply plugin: 'com.gradle.plugin-publish'
 apply from: "gradle/integration-test.gradle"
 
 codeQualityTools {
   ktlint {
-    toolVersion = '0.31.0'
+    toolVersion = '0.40.0'
   }
   detekt {
-    toolVersion = '1.0.0'
```

Comment:
Feel free to remove. I've stopped using detekt

Reply:
Will do that in a separate pr 👍